exercises/instrumentation/tracing/Dockerfile-client

```diff
@@ -0,0 +1,9 @@
+FROM python:3.9-alpine
+
+RUN apk update && apk upgrade && \
+    apk add --no-cache bash
+
+RUN pip install -r requirements.txt
```

Comment:
```suggestion
COPY requirements.txt requirements.txt
RUN pip install -r requirements.txt
```

Reply:
Hm, was that on purpose? 🤔 I'm not sure anymore 😅

Reply:
nah, my bad, fixed few other places as well